ledger/db/src/ledger_db.rs

```diff
@@ -900,6 +900,14 @@ impl LedgerDB {
     }
 }
 
+/// Creates a LedgerDB instance at the given path.
+pub fn create_ledger_in(path: &Path) -> LedgerDB {
+    let path = PathBuf::from(path);
+    let _ = std::fs::create_dir_all(&path);
```

Comment:
Any reason not to `.expect()` here?

Reply:
I thought this would fail if the dir exists, but they do check for that. Adding `expect`...

Reply:
You were right that we can `expect` since it does not return an error for existing dirs.